### PR TITLE
🎉 Add ability to save built app

### PR DESCRIPTION
**Pull request by albinekb**
Should we allow to build larger zips than 50 MB so you can debug if it's getting to large?
image

### CODE REVIEW

File: lib/tasks.js [hunk 1]
```diff
@@ -60,6 +62,18 @@ exports.packageApp = {
   }
 }
 
+exports.saveApp = {
+  title: 'Save packaged app',
+  enabled: (ctx) => Boolean(ctx.args['--output']),
+  task: async (ctx, task) => {
+    const { zipFile } = ctx
+    if (!zipFile) throw new Error('Missing ctx.zipFile')
+    const path = resolvePath(resolvePath(ctx.args['--output']))
```
Comment (LinusU) [inline lib/tasks.js]: Why are you calling `resolvePath` twice? 🤔

File: lib/builder.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ exports.createZipFile = async function (directory) {
   }
 
   const imageId = await execa.stdout('docker', ['build', '--quiet', '--file', 'scandium-dockerfile', '.'], { cwd: directory })
-  const zipFile = await execa.stdout('docker', ['run', '--rm', imageId], { cwd: directory, maxBuffer: 50000000 })
+  const zipFile = await execa.stdout('docker', ['run', '--rm', imageId], { cwd: directory, maxBuffer: 50000000 * 2 })
```
Comment (LinusU) [inline lib/builder.js]: 🤔

File: lib/tasks.js (hunk 1 above)
Comment (albinekb) [reply lib/tasks.js]: wops 😂

File: lib/builder.js (hunk 2 above)
Comment (albinekb) [reply lib/builder.js]: >Should we allow to build larger zips than 50 MB so you can debug if it's getting to large?


I'm thinking maybe allowing the buffer here to be 100 MB and then In the lambda upload check the size and not allow over 50 MB there? maybe even automatically save the file if it's larger than 50 MB to /tmp so you can inspect it?

Comment (LinusU) [reply lib/builder.js]: Set it to `100000000` then 😄, `* 2` looks very odd?

**Comment by albinekb** [comment]
How should we stop the file to try to be uploaded to lambda if it exceeds 50 MB?

**Comment by albinekb** [comment]
Since the zip is sent as base64 (very inefficient) from docker to scandium, it seems to exceed 50 MB, so bumping the max build size to 100 MB and doing the size check when uploading, we can still export and build zip files that exceed 50 MB for debugging purposes.

**Comment by LinusU** [comment]
Thanks! 🎉

**Pull request merged**